src/client/styles/modules/feed-layout.less

```diff
@@ -2,6 +2,8 @@
 
 .feed-layout {
   position: relative;
+  min-height: 100%;
```

Comment:
Is this supposed to fix the view being cut when feed page is loading? It doesn't seem to work for me

Reply:
I'm not sure how to solve this one. I think JM might know how to do this.

Reply:
This fix doesnt need to happen in this PR